Corrected the command to run sqlflow-all-in-one

MATRIX4284 opened the pull request:
Corrected the command to run sqlflow-all-in-one.yaml as it is wronlgly mentioned.

MATRIX4284 committed:
Corrected the command to run sqlflow-all-in-one

Corrected the command to run sqlflow-all-in-one.yaml as it is wronlgly mentioned.

wangkuiyi commented on doc/run/kubernetes.md:
```diff
@@ -21,11 +21,11 @@ to interact with the Kubernetes cluster.
 Docker image](../build.md).
 
-## Deploy the All-in-One SQLFlow
+## Deploy the All-in-One With MYSQL:
```
MYSQL => MySQL

wangkuiyi commented on doc/run/kubernetes.md:
```diff
@@ -21,11 +21,11 @@ to interact with the Kubernetes cluster.
+## Deploy the All-in-One With MYSQL:
 
-1. Deploy the SQLFlow Pod on Kubernetes
+1. Deploy the All-in-One SQLFLOW with MYSQL Pod on Kubernetes
```
All-in-one => all-in-one
SQLFLOW => SQLFlow

wangkuiyi reviewed (commented):
Some minor suggestions about the spell of popular nouns.

wangkuiyi committed:
Update kubernetes.md

As my above comments are minor, I did the change directly. Thanks for this change!

wangkuiyi committed:
Update kubernetes.md

MATRIX4284 commented:
Thanks a ton

On Wed, 8 Jan 2020 at 12:06 AM, Yi Wang <notifications@github.com> wrote:

> Merged #1631 <https://github.com/sql-machine-learning/sqlflow/pull/1631>
> into develop.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sql-machine-learning/sqlflow/pull/1631?email_source=notifications&email_token=AE7YQACMWOTPXLNEIAPBAQLQ4TDUFA5CNFSM4KDKSZ4KYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOV2HXKLA#event-2928637228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE7YQAFXQJ2OHAJIPN7DOMDQ4TDUFANCNFSM4KDKSZ4A>
> .
>
-- 
*Thanks & Regards,*
*Kaustav*
*Technical Envagelist*
*Embedded & Distributed Systems*
